desertbot/desertbot.py

```diff
@@ -117,12 +117,12 @@ def irc_JOIN(self, prefix, params):
 
         message = IRCMessage('JOIN', prefix, channel, u'', self)
 
-        if message.User.Name == self.nickname:
```

Comment:
These changes are not right, it's still User and ReplyTo in the IRCMessage class

Reply:
Changes from L120 - L268 in desertbot.py should be reverted back to point at User and ReplyTo for now.

Reply:
Oops. I actually tried to refactor those but reverted it to not cram too much into one PR. Looks like I didn't revert all of it...